plugins/notifications-backend-module-email/src/processor/transports/sendmail.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createTransport } from 'nodemailer';
+import { SendmailTransportConfig } from '../../types';
+
+export const createSendmailTransport = (config: SendmailTransportConfig) => {
```

Comment:
Could make sense to move all of the config reading into these functions as well; that'd alleviate the need for e.g. SendmailTransportConfig as a type to exist at all right? And localizes the functionality better. A bit much code in NotificationsEmailProcessor now that it itself isn't necessarily interested in knowing about, if i understand things right